source/max-staleness/max-staleness-tests.md

```diff
@@ -0,0 +1,142 @@
+# Max Staleness Tests
+
+These prose tests supplement the machine-readable YAML tests.
+
+## Connection string
+
+Tests of drivers' connection string parsing and validation.
+
+### Basic
+
+maxStalenessSeconds is set to 120 from this connection string:
+
+```
+mongodb://host/?readPreference=secondary&maxStalenessSeconds=120
+```
+
+### No maxStalenessSeconds with mode "primary"
+
+These connection strings are invalid because the mode is "primary", either by default or explicitly:
+
+```
+mongodb://host/?maxStalenessSeconds=120
+mongodb://host/?readPreference=primary&maxStalenessSeconds=120
+```
+
+### Specifying "no max staleness" with "-1"
+
+A MongoClient with this connection string has no max staleness, the same as if "maxStalenessSeconds" were omitted from
+the connection string:
+
+```
+mongodb://host/?readPreference=secondary&maxStalenessSeconds=-1
+```
+
+This connection string is valid and does not raise an error:
+
+> <mongodb://host/?readPreference=primary&maxStalenessSeconds=-1>
```

Comment:
Should this be in a code block?

````suggestion
```
mongodb://host/?readPreference=primary&maxStalenessSeconds=-1>
```
````